cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -489,3 +490,385 @@ def test_create_snapshot_via_ae(appliance, request, domain, small_test_vm):
 
     # Clean up if it appeared
     snapshot.delete()
+
+
+@pytest.mark.manual
+@pytest.mark.provider([InfraProvider])
+@test_requirements.ssui
+@test_requirements.snapshot
+@pytest.mark.tier(2)
+def test_sui_create_snapshot():
+    """
+    Snapshot can be created from VM details page, service details page
+    and snapshot page.
+    Check all pages and the snapshot count displayed on vm details page.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: SelfServiceUI
+        caseimportance: medium
+        initialEstimate: 1/4h
+        startsin: 5.9
+        title: SUI: Create snapshot from vm details page, snapshot page
+               and service details page
+        setup:
+            1. Add an infra provider
+            2. Create and order a service catalog which provisions a VM
+            3. Login to SSUI
+        testSteps:
+            1. Test that snapshot can be created from Service Details page
+                * Navigate to My Services -> <service_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            2. Test that snapshot can be created from VM Details page
+                * Navigate to My Services -> <service_name> -> <vm_name>
+                * Click Snapshots -> Create
+                * Fill Name and click Create
+            3. Test that snapshot can be created from Snapshot page
+                * Navigate to My Services -> <service_name> -> <vm_name> -> Snapshots
+                * Click Configuration -> Create Snapshot
+                * Fill Name and click Create
+        expectedResults:
+            1. Snapshot successfully created
+            2. Snapshot successfully created
+            3. Snapshot successfully created
+    """
+    pass
+
+
+@pytest.mark.manual
+@pytest.mark.provider([InfraProvider])
+@test_requirements.ssui
+@pytest.mark.tier(2)
+def test_snapshot_timeline_group_actions():
+    """
+    Test the SUI snapshot timeline.
+    Test grouping of actions in a timeline. Try to create a couple of
+    snapshots in a rapid succession, check how it looks in the timeline.
+
+    Polarion:
+        assignee: apagac
+        casecomponent: Infra
```

Comment:
Is this test for SUI ? If Yes then Case component will be SUI .

Reply:
casecomponents should be fixed now.